### ISSUE TITLE
Vector Animations with Python

### ISSUE THREAD

**Issue by stuaxo** [cairo-list/cairo-list#16]
http://zulko.github.io/blog/2014/09/20/vector-animations-with-python/

Based on cairocffi (which I believe we don't have... I'm not quite sure on the maintainership status of it)

Also need to add Qairah if we don't have it.

**Comment by rjopek** [comment]
Maybe a link to a specific file related to Cair?

**Comment by stuaxo** [comment]
Good point, maybe we could think about how to structure things a little based on the cairo bindings they use, for instance this one is based on `cairocffi` as it's binding, then it has `gizeh` provides nicer abstractions to draw shapes etc.
